driver/normalizer/parser.go

```diff
@@ -25,9 +25,25 @@ var ToNoder = &native.ObjectToNoder{
 	//      children node properties.
 }
 
+// transUASTParserBuilder creates a parser that transform source code files into *uast.Node
+// and calculates the offset from the position
+func transUASTParserBuilder(opts driver.UASTParserOptions) (tr driver.UASTParser, err error) {
```

Comment:
Yeah, I was trying to use the same name as the SDK struct (with a lowercase) but I like your proposal more.